test/symbol_name/nmcheck_prefix.pl

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/perl
```

Comment:
Probably better to

```sh
#!/usr/bin/env perl
```

this will let it find perl in the path.  Slightly more portable this way.